charts/pulsar/templates/pulsar-manager-configmap.yaml

```diff
@@ -27,5 +27,10 @@ metadata:
     {{- include "pulsar.standardLabels" . | nindent 4 }}
     component: {{ .Values.pulsar_manager.component }}
 data:
+  {{- if and (and .Values.auth.authorization.enabled .Values.auth.authentication.enabled) (eq .Values.auth.authentication.provider "jwt")}}
+  JWT_BROKER_TOKEN_MODE: PRIVATE
```

Comment:
pulasr-manager also supports another `SECRET` https://github.com/apache/pulsar-manager/tree/master/src#enable-jwt-auth mode, perhaps you can configure it as an optional parameter